src/models/Affiliation.ts

```diff
@@ -106,7 +105,7 @@ export class Affiliation extends MySqlModel {
     this.feedbackEmails = options.feedbackEmails;
 
     // We're proxying calls to funder APIs through the DMPHub API for now. This may change in the future
-    this.apiTarget = prepareAPITarget(options.apiTarget);
+    this.apiTarget = options.apiTarget; // prepareAPITarget(options.apiTarget);
```

Comment:
The .get method from DMPHubAPI already sets a base URL, so I thought that it made it easier to not add the DMPHub URL to apiTarget, but I'm not sure if this breaks how you are planning to use this for other things? So I could also remove the URL inside getAwards or before calling getAwards.